tests/use-cases/conftest.py

```diff
@@ -7,7 +7,7 @@
 
 FNAMES = [
     'pipeline.yaml',
-    'startscript',
+    'startscript.sh',
```

Comment:
need to agree on a common name for startscript files. ATM we were using just "startscript". better to have only one

Reply:
if this changes, you need to propagate the change to all the startscripts in the repo accordingly

Reply:
If it is currently startscript elsewhere, let us stick to that without '.sh'